server/views/pages/goal/update/index.njk

```diff
@@ -0,0 +1,161 @@
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageId = "update-goal" %}
+{% set pageTitle = applicationName + " - Home" %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Digital Prison Services",
+        href: dpsUrl
+      },
+      {
+        text: "Manage learning and work progress",
+        href: "/"
+      },
+      {
+        text: "Manage " + prisonerSummary.firstName | capitalize + " " + prisonerSummary.lastName | capitalize + "'s learning plan",
+        href: "/plan/" + prisonerSummary.prisonNumber + "/view/overview"
+      }
+    ],
+    classes: 'govuk-!-display-none-print'
+  }) }}
+{% endblock %}
+
+{% block content %}
+
+  {% if errors.length > 0 %}
+    {{ govukErrorSummary({
+      titleText: 'There is a problem',
+      errorList: errors,
+      attributes: { 'data-qa-errors': true }
+    }) }}
+  {% endif %}
+
+  {% include "../../../partials/prisonerBanner.njk" %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">You are updating a goal for {{ prisonerSummary.firstName | capitalize }} {{ prisonerSummary.lastName | capitalize }}</h1>
+
+        <form class="form" method="post" novalidate="">
+            <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+            <input type="hidden" name="prisonNumber" value="{{ form.prisonNumber }}" />
+            <input type="hidden" name="goalReference" value="{{ form.goalReference }}" />
```

Comment:
I think we'll need status as a hidden field as well (its part of the form, so has a value and needs to be submitted, but at the moment we dont allow the user to change the status ... that will come later)
Something like
```
<input type="hidden" name="status" value="{{ form.status }}" />
```